GitX.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>IDEDidComputeMac32BitWarning</key>
```

Comment:
I mean, arguably, this shouldn't be merged, but it doesn't really matter either.

Reply:
Before I ignored it in `.gitignore`, then I added it on request.
I want to avoid to ignore it manually by me locally.

I'm confused

Reply:
I'm just observing that this looks like a personal file that should be ignored since this is hiding a warning that might be interesting to users. @Drarok feels different. Neither of us is more right. It doesn't matter, but I'd prefer it being ignored until it has a critical file in it.

Reply:
Here's another idea–we can ignore JUST this file, and not the whole folder, so new files pop up.

Reply:
Seems like you _should_ commit this, at least according to [one StackOverflow user][1]. That would make sense given its location in `xcshareddata` – that's Apple's way of telling us.

[1]: https://stackoverflow.com/a/49564624/86093